UI/platform-windows.cpp

```diff
@@ -469,3 +474,26 @@ bool IsRunningOnWine()
 
 	return false;
 }
+
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
+QWinTaskbarButton *taskBtn;
+void TaskbarOverlayInit()
```

Comment:
If this PR also adds functionality to Windows, it should either be renamed or the changes decoupled into another PR.

Right now Jim (and I) were under the assumption this only adds functionality to macOS, but if it also changes other platforms I cannot fully review+merge this on my own.

Reply:
This functionality already exists for Windows. It was only moved from window-basic-main to the platform file, functionally was not changed.

Reply:
Ah right, you just moved the code around. Did you check the functionality on Windows?

Reply:
I downloaded CI builds on my fork and tested it still works for recordings